### ISSUE TITLE
making libreoffice fails

### ISSUE THREAD

**Issue by philipzae** [linuxscout/ayaspell#3]
When running `make libreoffice`, this is the output

#copy dic and aff files to OpenOffice/LibreOffice package
cp templates/ar.dic templates/openoffice/ar.dic
cp: cannot stat 'templates/ar.dic': No such file or directory
Makefile:63: recipe for target 'libreoffice' failed
make: *** [libreoffice] Error 1

**Comment by linuxscout** [comment]
÷ I fixed this error,can you confirm?
Thanks

**Comment by philipzae** [comment]
that part is now fixed and the zipped .oxt gets created, but as there is no releases folder, i get this error

...
mv templates/openoffice/dict_ar-3.7.oxt   releases/
mv: cannot move 'templates/openoffice/dict_ar-3.7.oxt' to 'releases/': Not a directory
Makefile:63: recipe for target 'libreoffice' failed
make: *** [libreoffice] Error 1

**Comment by mohataher** [comment]
`make libreoffice` still returns an error. 
```
$ make libreoffice
#copy dic and aff files to OpenOffice/LibreOffice package
cp templates/ar.dic templates/openoffice/ar.dic
cp templates/ar.aff templates/openoffice/ar.aff
#modify version
sed -i "s/<version value=\".*\"\/>/<version value=\"3.7\"\/>/g" templates/openoffice/description.xml 
sed: 1: "templates/openoffice/de ...": undefined label 'emplates/openoffice/description.xml'
make: *** [libreoffice] Error 1
```

**Comment by linuxscout** [comment]
Salam,
do you use a Mac?
I found a solution related to Mac on the following link:
https://stackoverflow.com/questions/23071830/sed-gives-sed-1-tsunit-js-undefined-label-sunit-js

**Comment by mohataher** [comment]
Yes, using Mac. I will test the solution when I get back to this task and let you know. Thanks.
